ssc/Pos/Ssc/Worker.hs

```diff
@@ -395,6 +397,8 @@ checkForIgnoredCommitmentsWorker
     => (WorkerSpec m, OutSpecs)
 checkForIgnoredCommitmentsWorker = localWorker $ do
     counter <- newTVarIO 0
+    -- TODO: [CSL-198] Catch and report non-critical errors, as
+    -- 'onNewSlot' no longer swallows all exception and rethrows them instead.
```

Comment:
I don't see why this worker may fail in normal cases, so let's stop program if it does (i. e. not catch anything here).